lib/workers/repository/init/merge.ts

```diff
@@ -49,12 +49,16 @@ export async function detectRepoFileConfig(): Promise<RepoFileConfig> {
   const cache = getCache();
   let { configFileName } = cache;
   if (configFileName) {
-    let configFileParsed = (await platform.getJsonFile(configFileName))!;
-    if (configFileParsed) {
-      if (configFileName === 'package.json') {
-        configFileParsed = configFileParsed.renovate;
+    try {
+      let configFileParsed = (await platform.getJsonFile(configFileName))!;
```

Comment:
```suggestion
      let configFileParsed = await platform.getJsonFile(configFileName);
```
is this related to this PR? should we extract to separate PR?

Reply:
Can be extracted to another PR 👍